src/utils/aotf.js

```diff
@@ -68,7 +69,8 @@ export const mutationIcons = {
   resume: mdiPlay,
   setOutputs: mdiGraph,
   stop: mdiStop,
-  trigger: mdiCursorPointer
+  trigger: mdiCursorPointer,
+  clean: mdiBroom
```

Comment:
Crept in to the wrong PR?

Besides, I think we might want to consider changing the icon for remove to e.g. `mdiMinusCircleOutline` and using the trash can icon `mdiDelete` or `mdiDeleteSweep` for clean

Reply:
It was deliberate - open to discussing changing it later - I'm not sure that the broom is obviously a broom - a trash can might be better. I considered a hoover and a spray bottle too!

Reply:
```suggestion
   trigger: mdiCursorPointer 
```